src/services/ledger/index.ts

```diff
@@ -24,19 +24,22 @@ import {
   toLiabilitiesWalletId,
   toWalletId,
 } from "@domain/ledger"
-import { lndAccountingPath, bankOwnerAccountPath } from "./accounts"
+import { lndAccountingPath, getBankOwnerWalletId } from "./accounts"
 
 type LoadLedgerParams = {
-  bankOwnerWalletResolver: () => Promise<string>
-  dealerWalletResolver: () => Promise<string>
+  bankOwnerWalletResolver: () => Promise<WalletId>
+  dealerWalletResolver: () => Promise<WalletId>
+  funderWalletResolver: () => Promise<WalletId>
 }
 
 export const loadLedger = ({
   bankOwnerWalletResolver,
   dealerWalletResolver,
+  funderWalletResolver,
 }: LoadLedgerParams) => {
   accounts.setbankOwnerWalletResolver(bankOwnerWalletResolver)
   accounts.setdealerWalletResolver(dealerWalletResolver)
+  accounts.setFunderWalletResolver(funderWalletResolver)
```

Comment:
Can probably make the camel-casing for these consistent as we're touching this code already here
```suggestion
  accounts.setBankOwnerWalletResolver(bankOwnerWalletResolver)
  accounts.setDealerWalletResolver(dealerWalletResolver)
  accounts.setFunderWalletResolver(funderWalletResolver)
```

Reply:
done